src/js/components/TimeSeriesChart/TimeSeriesChart.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import C3Wrapper from 'react-c3-wrapper';
+
+export default class TimeSeriesChart extends Component {
+
+  /**
+   * let xAxisValues = [{
+   *   'label': 'x',
+   *   'values': [1,2,3,4,5]
+   * },
+   * {
+   *   'label': 'cached',
+   *   'values: [1,2,3,4,5]
+   * }];
+   */
+  render() {
+    let { xAxisValues, yAxisLabel } = this.props;
+
+    let columns = [];
+
+    //First row of columms is the x axis key
+    //The first element in the remaining rows column has to be the label
+    for(let i = 0; i < xAxisValues.length; i++) {
+      if(xAxisValues[i].values) {
+        columns.push([xAxisValues[i].label].concat(xAxisValues[i].values));
```

Comment:
`columns.push([xAxisValues[i].label].concat(xAxisValues[i].values));`

It's hard to understand the paranthesis. Can we write it more explicit as in `xAxisValues[i].label` in variable etc. to make it easier to read